MekHQ/src/mekhq/campaign/mission/AtBContract.java

```diff
@@ -673,10 +673,12 @@ public void doBonusRoll(Campaign c) {
         switch (roll) {
         case 1: /* 1d6 dependents */
             number = Compute.d6();
-            c.addReport("Bonus: " + number + " dependent" + ((number>1)?"s":""));
-            for (int i = 0; i < number; i++) {
-                Person p = c.newDependent(Person.T_ASTECH, false);
-                c.recruitPerson(p, false, true, false, true);
+            c.addReport("Bonus: " + number + " dependent" + ((number > 1) ? "s" : ""));
```

Comment:
This should only be reported if you're actually adding them.

Reply:
That was outside to verify for testing, moved back inside so it is only reported in that case. Fixes for review changes have been pushed.